backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -0,0 +1,75 @@
+package cromwell.backend
+
+import akka.actor.{ActorRef, ActorLogging, Actor}
+import akka.event.LoggingReceive
+import cromwell.backend.WorkflowBackendActor._
+import cromwell.core.CallOutputs
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+import WorkflowBackendActor.ExecutionFailed
+
+object WorkflowBackendActor {
+
+  // Commands
+  sealed trait WorkflowBackendActorMessage
+
+  case class Execute(jobDescriptor: BackendJobDescriptor) extends WorkflowBackendActorMessage
+  case class Recover(jobDescriptor: BackendJobDescriptor) extends WorkflowBackendActorMessage
+  case class Abort(jobKey: BackendJobDescriptorKey) extends WorkflowBackendActorMessage
+
+  // Responses
+  sealed trait WorkflowBackendActorResponse
+
+  sealed trait ExecutionResponse extends WorkflowBackendActorResponse
+  case class ExecutionSucceeded(jobKey: BackendJobDescriptorKey, callOutputs: CallOutputs) extends ExecutionResponse
+  object ExecutionFailed {
+    def make(backendJobDescriptor: BackendJobDescriptor, throwable: Throwable) = ExecutionFailed(backendJobDescriptor.key, throwable)
```

Comment:
Ah okay, I (finally) get it.  Maybe something that makes this more clear in case others are slow like I was?  😄 

```scala
 private def descriptorToFailure: (BackendJobDescriptor, Throwable) => ExecutionFailed =
   (jd, t) => ExecutionFailed(jd.key, t)
```